go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -184,6 +186,35 @@ func initMySQLProtocol() {
 	}
 }
 
+// newtMysqlUnixSocket creates a new unix socket mysql listener. If a socket file already exists, attempts
+// to clean it up.
+func newMysqlUnixSocket(address string, authServer mysql.AuthServer, handler mysql.Handler) (*mysql.Listener, error) {
+	mysqlUnixListener, err := mysql.NewListener("unix", address, authServer, handler)
```

Comment:
nit: Here and below: `mysqlUnixListener` is a mouth full. From the context and scope of the variable it is clear, that it can only be one kind of listener. Hence I recommend to rename it to `listener`.

Reply:
Sorry, I wasn't clear enough here: I meant the variable and not the method name :)

The method name is actually great. Since the scope of the method is the "vtgate" package, it makes sense to be specific here.

Do you mind to revert back to "newMysqlUnixSocket" as method name and change the variable name here and throughout the file? i.e.

```go
// newMysqlUnixSocket creates a new unix socket mysql listener. If a socket file already exists, attempts
// to clean it up.
func newMysqlUnixSocket(address string, authServer mysql.AuthServer, handler mysql.Handler) (*mysql.Listener, error) {
  listener, err := mysql.NewListener("unix", address, authServer, handler)
...
```